prepline_general/api/general.py

```diff
@@ -295,7 +295,7 @@ def pipeline_api(
     hi_res_model_name: Optional[str] = None,
     include_page_breaks: bool = False,
     ocr_languages: Optional[List[str]] = None,
-    pdf_infer_table_structure: bool = True,
+    pdf_infer_table_structure: bool = False,
```

Comment:
@awalker4 - Before I merge here, could you check my math on this? Had to make this change for `scripts/smoketest.py` to pass because the default in `unstructured` changed to "table extraction off for PDFs and images". With this update, the default API behavior doesn't change for `skip_infer_table_types`, which is now the supported parameter for changing table behavior. It does change for `pdf_infer_table_structure`, which we've been emitting deprecation warnings about for a while. Does that make sense?

Reply:
Disregard this comment